nmdc_automation/workflow_automation/watch_nmdc.py

```diff
@@ -14,7 +14,17 @@
 
 logger = logging.getLogger(__name__)
 
-
+# TODO: Berkley refactoring:
+#   The watcher interacts with the NMDC runtime API to find / claim jobs and to post the resulting
```

Comment:
any code that uses an endpoint that references activity is will change, see https://github.com/microbiomedata/nmdc-runtime/issues/593 for the new endpoint names. There is a linked draft PR which is not merged yet.